.github/workflows/publish.yaml

```
@@ -32,7 +34,13 @@ jobs:
       branch: ${{ github.event.inputs.branch }}
       environment: ${{ github.event.inputs.environment }}
       docs-only: ${{ fromJSON(github.event.inputs.docs-only) }}
-      # Disable playwright tests for now
-      run-playwright: false
+      scopes: universal
+      
+      # Argo CD inputs
       grafana-cloud-deployment-type: provisioned
-      argo-workflow-slack-channel: '#grafana-plugins-platform-ci'
+      argo-workflow-slack-channel: "#grafana-plugins-platform-ci"
+
+      # Other CI inputs
+      # Disable playwright tests for now
+      run-playwright: false
```

Let's enable these tests now

```suggestion
      run-playwright: true
```